server/liveblog/syndication/utils.py

```diff
@@ -96,3 +100,148 @@ def fetch_url(url, timeout=5):
             fd.write(chunk)
     fd.seek(0)
     return fd
+
+
+def extract_post_items(original_doc):
```

Comment:
I actually think that this should be in syndication.py, as it extracts just the data which is required to send to the consumer via the webhook, and not the items themselves. Perhaps it should be renamed extract_post_items_data